### PR TITLE
Ops 1879 : Remove new containers that fail

### PR DESCRIPTION
**Pull request by chrisd0**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -129,6 +129,13 @@ func (this *DockerClient) onContainerStarted(id string) bool {
 		for {
 			if time.Now().After(end) {
 				log.Printf("New container came online, but did not respond to status queries.")
+				log.Printf("Killing new container: %s", id)
+                        	err := this.dc.KillContainer(docker.KillContainerOptions{
+                     	          ID: id,
```
Comment (jpverkamp) [inline main.go]: Odd formatting. Try `go fmt`.

File: main.go [hunk 2]
```diff
@@ -129,6 +129,13 @@ func (this *DockerClient) onContainerStarted(id string) bool {
...
+                       		})
+                      		if err != nil {
+                               		log.Printf("Failed to signal container: %s", err.Error())
```
Comment (jpverkamp) [inline main.go]: Odd formatting (indentation), `go fmt`.

**Comment by jpverkamp** [comment]
Depends on https://github.com/edmodo/docker-proxy/pull/2 being merged first. 

Test case:
- Set timeout to 5s
- Start regular server (A)
- Start server on 20s delay (B)
- Verify that A is still running
- Verify that B is no longer running

This test case does not work on master, but does work on the new branch.

✨

**Comment by chrisd0** [comment]
Yep, the whitespace changes look fine.  I think it is good to go.  We should just verify that #2 is ready, and then try a merge / update.

**Pull request merged**